### DISCUSSION TITLE
File System Drop Box Support

### DISCUSSION THREAD

**Discussion by DustinBrett** [DustinBrett/daedalOS#85] [Feature Requests]
This is part of BrowserFS already but I have not added it to my file system yet.

**Comment by hello-smile6** [comment]
Could you do Google Drive?

**Reply by DustinBrett** [reply]
I'm not sure tbh. It seems less likely with their API's. But maybe one day. Feel free to do some research on it. :-)

**Reply by hello-smile6** [reply]
I'd like Google Drive because then I could `git clone` repositories on a remote computer via VNC and then use Git locally using the filesystem mount feature to make changes.

**Comment by DustinBrett** [comment]
Looked into the stuff built into BrowserFS and it seems to be an old API. I am still interested in doing this or IPFS or both, but it might be more work.
